crates/fuel-core/src/graphql_api/api_service.rs

```diff
@@ -221,17 +221,26 @@ where
         .extension(ViewExtension::new())
         .finish();
 
-    let router = Router::new()
-        .route("/v1/playground", get(graphql_playground))
-        .route("/v1/graphql", post(graphql_handler).options(ok))
-        .route(
-            "/v1/graphql-sub",
-            post(graphql_subscription_handler).options(ok),
-        )
-        .route("/v1/metrics", get(metrics))
+    let unthrottled_routes = Router::new()
+        // we shouldn't throttle these requests
```

Comment:
Nit: I think the variable name is clear enough here.
```suggestion
```

Reply:
addressed in aa651b81a6